src/content/blog/astro-3.mdx

```diff
@@ -111,7 +111,7 @@ Astro users have had early access to View Transitions since Astro 2.9. Our commu
 
 We've put a ton of effort in to make View Transitions work great across all browsers, even those that [don't yet support](https://caniuse.com/view-transitions) the native View Transition APIs. Astro 3.0 includes fallbacks for most browsers, in the form of a small, ~3kb script that Astro adds to your page automatically.
 
-Learn more about View Transitions in our handy [View Transitions guide](https://docs.astro.build/en/guides/view-transitions/), or [this great article](https://developer.chrome.com/docs/web-platform/view-transitions/)  about the browser APIs by the Chrome team.
+Learn more about View Transitions in our handy [View Transitions guide](https://docs.astro.build/en/guides/view-transitions/), or [this great article](https://developer.chrome.com/docs/web-platform/view-transitions/)  about the browser APIs by the Chrome team. Also read more on our journey to adopt View Transitions on the [Chrome Developer blog](https://developer.chrome.com/blog/astro-view-transitions/).
```

Comment:
```suggestion
Learn more about View Transitions in our handy [View Transitions guide](https://docs.astro.build/en/guides/view-transitions/), or [this great article about the browser APIs](https://developer.chrome.com/docs/web-platform/view-transitions/)  by the Chrome team. Also read more about our journey to adopt View Transitions on the [Chrome Developer blog](https://developer.chrome.com/blog/astro-view-transitions/).
```
Just suggesting making the first link text, more descriptive for good a11y. (And "about" so that we don't have to say "on" twice so close together.)